src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -310,6 +310,10 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 			this._startBlockingIframeDragEvents();
 		}));
 
+		this._register(this.on('drag', (event) => {
```

Comment:
these changes allow events from the iframes to bubble up to the standard VSCode webviewWindowDragAndDropMonitor